themes/10up-block-theme/assets/css/base/layout.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * These are deprecated fallbacks for the variables that now live in the
+ * theme.json file. They are here to ensure backwards compatibility with
+ * older themes that do used these variables in their styles.
+ */
+:root {
+	--full-viewport-width: var(--wp--custom--full-viewport-width);
+	--site-outer-padding: var(--wp--custom--site-outer-padding);
+	--site-content-width: var(--wp--custom--site-content-width);
+	--main-content-width-side-spacing: var(--wp--custom--main-content-width-side-spacing);
+	--main-wide-width-side-spacing: var(--wp--custom--main-wide-width-side-spacing);
+	/* stylelint-disable-next-line length-zero-no-unit */
+	--site-header-top-offset: var(--wp-admin--admin-bar--height, 0px);
+
+	&.has-pinned-header {
```

Comment:
Is `has-pinned-header` a class generated by WP in any way?  This seems a bit unnecessary unless there is some mechanism by which that would be commonly added in addition to the `--header-height` var.

Reply:
Whoops, yeah these should be removed. They are project specific